src/Source.h

```diff
@@ -103,10 +171,20 @@ class Source {
   static SourcePtr create(Rcpp::List spec);
 
 private:
-  static bool
-  inComment(const char* cur, const char* end, const std::string& comment) {
-    boost::iterator_range<const char*> haystack(cur, end);
-    return boost::starts_with(haystack, comment);
+  std::string pEncoding;
```

Comment:
I don't think our style prefixes private variables. @jimhester can you please confirm?